samigo/samigo-import/src/java/org/sakaiproject/importer/impl/handlers/SamigoAssessmentHandler.java

```diff
@@ -136,11 +136,9 @@ public int compare(Object o1, Object o2) {
 			assessment.setSectionSet(sectionSet);
 			as.saveAssessment(assessment);
 		} catch (Exception e) {
-			// error creating this assessment
-			log.error(e.getMessage(), e);
+			log.warn("Error creating assessment: {}", e.toString());
 		} catch (AssessmentException e) {
```

Comment:
given your catching Exception first this will never be reached, since they both do the same I would just remove this one.

Reply:
You still have to catch it. AssessmentException must not subclass Exception.